clang/lib/Basic/OpenMPKinds.cpp

```diff
@@ -608,6 +617,16 @@ const char *clang::getOpenMPSimpleClauseTypeName(OpenMPClauseKind Kind,
 #include "clang/Basic/OpenMPKinds.def"
     }
     llvm_unreachable("Invalid OpenMP 'threadset' clause modifier");
```

Comment:
We should update this to say `argument` as well, but that's not related to this PR.